code/game/objects/items/weapons/storage/secure.dm

```diff
@@ -228,7 +228,7 @@
 //        Secure Safe
 // -----------------------------
 
-/obj/item/weapon/storage/secure/safe
+/obj/item/weapon/storage/secure/safe//Why the fuck isn't this a structure
```

Comment:
Why isn't it a structure though? If possible I'd rather turn it into a structure here and now unless someone gives me a good enough reason.